docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/example03.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Non-compliant Code Example """
+import locale
+ORIGINAL_NUMBER = 12.345  # This will read as 12,345 in German
+ 
+def compare_number(number):
```

Comment:
Getting this from Flake:
expected 2 blank lines, found 1Flake8(E302)
Missing function or method docstringPylint[C0116:missing-function-docstring](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-function-docstring.html)